config/clusters/callysto/common.values.yaml

```diff
@@ -65,67 +65,68 @@ jupyterhub:
   hub:
     extraConfig:
       001-cilogon-email-auth: |
-        # Custom override of CILogonOAuthenticator to allow access control via one
-        # property (email) while the username itself is a different property (oidc).
-        # This allows us to restrict access based on the *email* of the authenticated
-        # user without having to store the actual email in our servers.
-        from oauthenticator.cilogon import CILogonOAuthenticator
         from fnmatch import fnmatch
-        from traitlets import List, Unicode
+
+        from oauthenticator.cilogon import CILogonOAuthenticator
+
 
         class EmailAuthenticatingCILogonOAuthenticator(CILogonOAuthenticator):
-          allowed_domains = List(
-            Unicode,
-            default=None,
-            # This must always be set!
-            allow_none=False,
-            help="""
-            Allow access to users with verified emails belonging to these domains.
+            """
+            Custom override of CILogonOAuthenticator to allow access control via
+            one property (email) while the username itself is a different
+            property (oidc), and to allow `allowed_domains` to involve wildcards
+            like *.
+
+            This allows us to restrict access based on the *email* of the
+            authenticated user without having to store the actual email in our
+            servers or use it as a username.
+            """
 
-            Allows glob patterns but not regexes. So you can allow access to
-            *.edu, for example. See https://docs.python.org/3/library/fnmatch.html for
-            list of allowed patterns.
-            """,
-            config=True
-          )
```

Comment:
Instead of defining a new trait, we rely on the existing on provided under the individual `allowed_idps` entries.